pkg/ovirt/manifest.json.in

```diff
@@ -0,0 +1,16 @@
+{
+  "version": 0,
+  "name": "machines",
+  "priority": 10,
+  "requires": {
+       "cockpit": "122"
+  },
+  "menu": {
+     "vms": {
+        "label": "oVirt Machines",
+        "path": "index.html",
+        "order": 60
+     }
+  },
+  "content-security-policy": "default-src 'self' data:"
```

Comment:
We shouldn't need to include content-security-policy for any of our default components. Is this really necessary?

Reply:
Right, since cockpit-ovirt `requires` post-installation step to generate config file and override.json, this is not needed. So I removed this from manifest.json.